packages/components/pie-cookie-banner/src/cookie-banner.scss

```diff
@@ -103,22 +98,32 @@ $breakpoint-wide: 700px;
     display: flex;
     gap: var(--dt-spacing-d);
     flex-direction: var(--cb-actions-flex-dir);
-
-    > pie-link {
-        width: max-content;
-        margin-inline-start: auto;
-        margin-inline-end: auto;
-    }
```

Comment:
are we happy with pie-link being full-width? It now means there is invisible clicking areas (which might be ok but worth checking)

<img width="436" alt="Screenshot 2023-08-31 at 15 38 57" src="https://github.com/justeattakeaway/pie/assets/16766185/879956c5-5740-45a5-b46f-3fce3f69764e">